client/my-sites/domains/domain-management/domain-connect/domain-connect-authorize.jsx

```diff
@@ -66,44 +66,48 @@ class DomainConnectAuthorize extends Component {
 
 	handleClickConfirm = () => {
 		const { providerId, serviceId, params, translate } = this.props;
-		const { domain } = params;
```

Comment:
Confirming that we don't need to pass the domain to the endpoint route, it's already passed as a query argument as part of the `params`. So that's fine and expected by the endpoint.